### PR TITLE
remove log.error() on cases where it's not really system error

### PR DESCRIPTION
**Pull request by shintasmith**
remove calls to log.error() as an invalid user input should not be considered an error

### CODE REVIEW

**Comment by shintasmith** [comment]
@usnavi @ChandraAddala @izrik pls review

File: blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedMultiIngestionHandler.java [hunk 1]
```diff
@@ -139,16 +139,14 @@ public void handle(ChannelHandlerContext ctx, FullHttpRequest request) {
...
             DefaultHandler.sendErrorResponse(ctx, request, ex.getMessage(), HttpResponseStatus.BAD_REQUEST);
         } catch (InvalidDataException ex) {
             log.debug(String.format("Invalid request body: %s", body));
-            log.error(ex.getMessage(), ex);
             DefaultHandler.sendErrorResponse(ctx, request, ex.getMessage(), HttpResponseStatus.BAD_REQUEST);
         } catch (TimeoutException ex) {
             DefaultHandler.sendErrorResponse(ctx, request, "Timed out persisting metrics", HttpResponseStatus.ACCEPTED);
         } catch (Exception ex) {
-            log.debug(String.format("BAD JSON: %s", body));
+            log.debug(String.format("Exception processing: %s", body));
```
Comment (ChandraAddala) [inline blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedMultiIngestionHandler.java]: If we get some exception which we are not expecting, I think we need some visibility to it. Should we log error here?

Comment (shintasmith) [inline blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedMultiIngestionHandler.java]: There is already a log.error(). See the line below.

Comment (ChandraAddala) [inline blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedMultiIngestionHandler.java]: I must be blind.

**Pull request merged**